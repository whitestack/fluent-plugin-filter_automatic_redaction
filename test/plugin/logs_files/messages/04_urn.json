[
  {
    "input_message": "Lorem Ipsum urn:whitestack:whitecloud Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum urn:whitestack:whitestorage Lorem Ipsum"
  }
]