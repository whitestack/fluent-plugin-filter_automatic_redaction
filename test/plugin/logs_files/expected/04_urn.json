[
  {
    "input_message": "Lorem Ipsum urn:whitestack:whitecloud Lorem Ipsum",
    "output_message": "Lorem Ipsum <URN> Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum urn:whitestack:whitestorage Lorem Ipsum",
    "output_message": "Lorem Ipsum <URN> Lorem Ipsum"
  }
]